### DISCUSSION TITLE
Pest `not()` Method to `Str` and `str()`

### DISCUSSION THREAD

**Discussion by devajmeireles** [laravel/framework#48308] [Ideas]
Here is one more suggestion that would bring some benefit to Laravel, this time based on what we have in [Pest](https://pestphp.com/docs/expectations#expect-not):

![image](https://github.com/laravel/framework/assets/60591772/9ecc2a3e-6de2-47b2-bec4-39843fd5cd16)

The addition of the `not` method to inverse the result of the comparison of all `str` boolean methods:
- is 
- contains / containsAll
- isMatch / startsWith / endsWith

And others.

**Comment by henzeb** [comment]
Isn't it easier to have a simple negate method instead? 'isNot', 'misses', 'notMatches' etc?

**Reply by joelharkes** [reply]
indeed laravel tries to make it as readable as possible `->doesNotContain()` is much more readable in english.
